internal/characters/itto/charge.go

```diff
@@ -173,7 +175,8 @@ func (c *char) ChargeAttack(p map[string]int) action.ActionInfo {
 
 	prevSlash := c.slashState
 	stacks := c.Tags[strStackKey]
-	c.slashState = prevSlash.Next(stacks)
+	hasC6 := c.Base.Cons >= 6 && c.Core.Rand.Float64() < 0.5
+	c.slashState = prevSlash.Next(stacks, hasC6)
```

Comment:
I think `hasC6` as a name is misleading, as this variable is only true half the time. Do you think something like `didC6Proc` would be more readable?

Reply:
How about `c6Proc`?

Reply:
I'm okay with that.